frontend/app/application.py

```diff
@@ -245,7 +245,7 @@ def get_year_from_date_string(date_string):
 
 @application.route('/projects/')
 def project_index_template():
-    url = api_url + "/project_cache"
+    url = api_url + "/project_cache?isPublished=true"
```

Comment:
```suggestion
    url = api_url + "/project_cache"
```

This is not needed because `software_cahce` only contains items where `isPubihsed` is `true` in the `software` collection.